exec/java-exec/src/main/java/org/apache/drill/exec/server/options/FallbackOptionManager.java

```diff
@@ -83,14 +84,30 @@ public OptionValue getOption(final String name) {
    */
   abstract boolean setLocalOption(OptionValue value);
 
+  /**
+   * Deletes the option with given name for this manager without falling back.
+   *
+   * @param type option type
+   * @return true iff the option was successfully deleted
```

Comment:
actually it's less confusing here because we only pass an `OptionType` to the method but it's more prevalent for `deleteLocalOption()`